site/el/README.md

```diff
@@ -0,0 +1,20 @@
+# Μετάφραση κοινότητας
+
+Η κοινότητα του TensorFlow έχει μεταφράσει αυτά τα έγγραφα. Καθότι οι μεταφράσεις αυτές αποτελούν την καλύτερη δυνατή προσπάθεια , δεν υπάρχει εγγύηση ότι θα παραμείνουν ενημερωμένες σε σχέση με τα [επίσημα Αγγλικά έγγραφα](https://www.tensorflow.org/?hl=en).
+Αν έχετε υποδείξεις για βελτίωση των αρχείων αυτών , δημιουργήστε ένα pull request στο [tensorflow/docs](https://github.com/tensorflow/docs) GitHub repository . Για να συμμετέχετε στη σύνταξη ή στην αναθεώρηση των μεταφράσεων της κοινότητας , επικοινωνήστε με το [docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
+
+# Community translations
+
+Our TensorFlow community has translated these documents. Because community
+translations are *best-effort*, there is no guarantee that this is an accurate
+and up-to-date reflection of the
+[official English documentation](https://www.tensorflow.org/?hl=en).
+If you have suggestions to improve this translation, please send a pull request
+to the [tensorflow/docs](https://github.com/tensorflow/docs) GitHub repository.
+To volunteer to write or review community translations, contact the
+[docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
+
+# Προς τον Έλληνα αναγνώστη
+
+Στα πλαίσια μίας εργασίας για το πανεπιστήμιο, ξεκίνησα αυτό το εγχείρημα. Δεν έβγαλα πολλές μεταφράσεις, ωστόσο θα επιστρέψω για να τις ολοκληρώσω (κάποια στιγμή..). Αν έτυχε να το διαβάζεις αυτό , και αν σου άρεσε το αποτέλεσμα , μπορείς να συμμετέχεις.
+Αρκετή ορολογία έμεινε ως έχει για να διατηρηθεί,κατά τη γνώμη μου , η ροή του λόγου.
```

Comment:
Thanks.
Consider adding a `## Do not translate` section at the end of this file. Most languages have this since machine learning terminology is so new, effective translations do not yet exist. For example, here's the Italian list: https://github.com/tensorflow/docs-l10n/tree/master/site/it#guida-alla-traduzione-italiana